src/Bridges/Constraint/bridges/fix_parametric_variables.jl

```diff
@@ -0,0 +1,161 @@
+    values = Dict{MOI.VariableIndex,Union{Nothing,T}}()
+    new_coefs = Dict{MOI.VariableIndex,T}()
+    for term in f.quadratic_terms
+        values[term.variable_1] = nothing
+        values[term.variable_2] = nothing
+        new_coefs[term.variable_1] = zero(T)
+        new_coefs[term.variable_2] = zero(T)
+    end
+    return FixParametricVariablesBridge{T,S}(ci, f, values, new_coefs)
+end
+
+function MOI.supports_constraint(
+    ::Type{<:FixParametricVariablesBridge{T}},
+    ::Type{MOI.ScalarQuadraticFunction{T}},
+    ::Type{<:MOI.AbstractScalarSet},
+) where {T}
+    return true
+end
+
+function concrete_bridge_type(
+    ::Type{<:FixParametricVariablesBridge},
+    ::Type{MOI.ScalarQuadraticFunction{T}},
+    ::Type{S},
+) where {T,S<:MOI.AbstractScalarSet}
+    return FixParametricVariablesBridge{T,S}
+end
+
+function MOI.Bridges.added_constrained_variable_types(
+    ::Type{<:FixParametricVariablesBridge},
+)
+    return Tuple{Type}[]
+end
+
+function MOI.Bridges.added_constraint_types(
+    ::Type{FixParametricVariablesBridge{T,S}},
+) where {T,S}
+    return Tuple{Type,Type}[(MOI.ScalarAffineFunction{T}, S)]
+end
+
+function MOI.get(
+    ::MOI.ModelLike,
+    ::MOI.ConstraintFunction,
+    bridge::FixParametricVariablesBridge,
+)
+    return bridge.f
+end
+
+function MOI.get(
+    model::MOI.ModelLike,
+    ::MOI.ConstraintSet,
+    bridge::FixParametricVariablesBridge,
+)
+    return MOI.get(model, MOI.ConstraintSet(), bridge.affine_constraint)
+end
+
+function MOI.delete(model::MOI.ModelLike, bridge::FixParametricVariablesBridge)
+    MOI.delete(model, bridge.affine_constraint)
+    return
+end
+
+MOI.get(::FixParametricVariablesBridge, ::MOI.NumberOfVariables)::Int64 = 0
+
+function MOI.get(::FixParametricVariablesBridge, ::MOI.ListOfVariableIndices)
+    return MOI.VariableIndex[]
+end
+
+function MOI.get(
+    bridge::FixParametricVariablesBridge{T,S},
+    ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T},S},
+)::Int64 where {T,S}
+    return 1
+end
+
+function MOI.get(
+    bridge::FixParametricVariablesBridge{T,S},
+    ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{T},S},
+) where {T,S}
+    return [bridge.affine_constraint]
+end
+
+function MOI.modify(
+    model::MOI.ModelLike,
+    bridge::FixParametricVariablesBridge{T,S},
+    chg::MOI.ScalarCoefficientChange{T},
+) where {T,S}
+    MOI.modify(model, bridge.affine_constraint, chg)
+    MOI.Utilities.modify_function!(bridge.f, chg)
+    return
+end
+
+MOI.Bridges.needs_final_touch(::FixParametricVariablesBridge) = true
+
+function MOI.Bridges.final_touch(
+    bridge::FixParametricVariablesBridge{T,S},
+    model::MOI.ModelLike,
+) where {T,S}
+    for x in keys(bridge.values)
+        ci = MOI.ConstraintIndex{MOI.VariableIndex,MOI.EqualTo{T}}(x.value)
+        if MOI.is_valid(model, ci)
+            bridge.values[x] = MOI.get(model, MOI.ConstraintSet(), ci).value
+        else
+            bridge.values[x] = nothing
```

Comment:
Add `bridge.new_coefs[x] = nothing` in case the variable was fixed in a previous `optimize!` but it is not anymore